### ISSUE TITLE
 Can you provide a pretrain model?

### ISSUE THREAD

**Issue by Rhett98** [mareksubocz/TrackNet#3]
I can't get a good dataset with label. If you can provide a pretrain model, i'd gratitude it

**Comment by mareksubocz** [comment]
Sure, I'll provide a model pretrained on volleyball videos, but I'm not sure if it will be good for any other use case. Can you share what is your intended use and if you plan on finetuning it? No problem if you can't, just here to help.

**Comment by Rhett98** [comment]
这是来自QQ邮箱的自动回复邮件。你好，我已收到你的邮件，我查看后会尽快给你回复。

**Comment by fioresxcat** [comment]
Does the model work well on your volleyball videos ? I'm intended to use it for table tennis. Really happy if you could confirm that.

**Comment by Rhett98** [comment]
这是来自QQ邮箱的自动回复邮件。你好，我已收到你的邮件，我查看后会尽快给你回复。

**Comment by mareksubocz** [comment]
> Does the model work well on your volleyball videos ? I'm intended to use it for table tennis. Really happy if you could confirm that.

It definitely detects the ball well, sometimes even when it's not visible for a frame, but it turns out that players' heads look really similar to a volleyball ball and there are some false positives. For my use case I need as little amount of false positives as possible and don't mind not detecting the ball sometimes, and YOLOv8 works better in that case.

EDIT: one thing I forgot is that it sometimes struggle when the ball is stationary, as it's a movement-based approach. It happens in volleyball when the ball is at the top of its curve, but I assume it's not really an issue for table tennis.

**Comment by fioresxcat** [comment]
Thank you for your answer! I'm using YOLOv8 as well and it still has quite a lot false positive where other similar things are misclasssified as ball. My use case also needs as few false positives as possible. Have you done anything to address that issue, or do you suggest any possible solution ?

**Comment by mareksubocz** [comment]
I think in your case the false positive come from the fact that a table tennis ball is just a small white blob and a lot of things are. In that case, tracknet could actually have less false positives than YOLO. Volleyball ball is colorful and that's probabaly why a purely image-based YOLO is better.

**Comment by fioresxcat** [comment]
I'll give it a try. Thanks a lot for sharing your code

**Comment by pantongwang** [comment]
Can you provide a pretrain model for me too? pls, I will appreciate.

**Comment by Rhett98** [comment]
这是来自QQ邮箱的自动回复邮件。你好，我已收到你的邮件，我查看后会尽快给你回复。

**Comment by 121649982** [comment]
can you give me a pretrain model too?thank you   ! email:121649982@qq.com

**Comment by Rhett98** [comment]
这是来自QQ邮箱的自动回复邮件。你好，我已收到你的邮件，我查看后会尽快给你回复。

**Comment by mareksubocz** [comment]
I unfortunately lost the pretrained model on my old machine :( i'll happily include it in repo if somebody decides to train it, as well as accept any reasonable pull requests.
